packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -946,6 +946,21 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     return binding.takeException();
   }
 
+  /// Returns a list of all the accessibility announcements made by the Flutter
+  /// framework since the last time this function was called.
+  ///
+  /// See [TestWidgetsFlutterBinding.takeAnnouncements] for details.
+  List<CapturedAccessibilityAnnouncement>? takeAnnouncements() {
+    return binding.takeAnnouncements();
+  }
+
+  /// Returns the most recent announcement made by the Flutter framework.
+  ///
+  /// See [TestWidgetsFlutterBinding.peekLastAnnouncement] for details.
+  CapturedAccessibilityAnnouncement? peekLastAnnouncement() {
```

Comment:
Do we need both of these APIs? Wouldn't it be enough to just have the takeAnnouncements one? If we go with both, the API docs should link to the other one in a "see also" section.

Reply:
I agree. Removed peekLastAnnouncement() since having the `takeAnnouncements()` seems enough.